### PR TITLE
flow-related recipe

### PR DESCRIPTION
**Pull request by cristhomas**
* [x] I have read the guidelines above.
* [x] This PR adds several new recipe.
@bioconda/core

### CODE REVIEW

File: recipes/r-mba/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+package:
+  name: r-mba
+  # Note that conda versions cannot contain -, so any -'s in the version have
+  # been replaced with _'s.
+  version: "0.0_8"
+
+source:
+  fn: MBA_0.0-8.tar.gz
+  url:
+    - http://cran.r-project.org/src/contrib/MBA_0.0-8.tar.gz
```
Comment (bgruening) [inline recipes/r-mba/meta.yaml]: can you please add checksums here

File: recipes/r-rfoc/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+package:
+  name: r-rfoc
+  # Note that conda versions cannot contain -, so any -'s in the version have
+  # been replaced with _'s.
+  version: "3.3_3"
+
+source:
+  fn: RFOC_3.3-3.tar.gz
+  url:
+    - http://cran.r-project.org/src/contrib/RFOC_3.3-3.tar.gz
```
Comment (bgruening) [inline recipes/r-rfoc/meta.yaml]: Checksums?

File: recipes/r-robustbase/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+package:
+  name: r-robustbase
+  # Note that conda versions cannot contain -, so any -'s in the version have
+  # been replaced with _'s.
+  version: "0.92_6"
+
+source:
+  fn: robustbase_0.92-6.tar.gz
+  url:
+    - http://cran.r-project.org/src/contrib/robustbase_0.92-6.tar.gz
```
Comment (bgruening) [inline recipes/r-robustbase/meta.yaml]: I'm wondering why these checksums are not automatically included ...

**Comment by bgruening** [comment]
What a awesome contribution! Thanks a lot!
@cristhomas most of the other packages uses `r` and not `r-base` maybe you can change this? And feel free to remove some of the comments in your file.

**Comment by bgruening** [comment]
`ERROR: dependency ‘BH’ is not available for package ‘flowCore’` do you know what BH is?

**Comment by cristhomas** [comment]
That's odd about the checksum, I didn't think to check because they were included on the recipe I generated yesterday. I'll do that now. Also the 'BH' thing I don't know. It worked fine on my machine, I'll look into it.

File: recipes/r-robustbase/meta.yaml (hunk 3 above)
Comment (cristhomas) [inline recipes/r-robustbase/meta.yaml]: so looks like other recipe for r packages in the existing bioconda recipe don't have md5 either. I wonder if it just doesn't include them automatically when you build your recipe with 'conda skeleton'?

Comment (bgruening) [inline recipes/r-robustbase/meta.yaml]: I think so and we should fix this. Sorry to bother you, it's fine. Leave it like it is for now.

Comment (cristhomas) [inline recipes/r-robustbase/meta.yaml]: that's ok, i fixed the files already =)

**Comment by cristhomas** [comment]
so how did you get the 'BH' dependency error?

**Comment by bgruening** [comment]
Oh is it this one? https://cran.r-project.org/web/packages/BH/index.html

**Comment by cristhomas** [comment]
it probably is yes, if it's not installed for you - it's installed on my system.

**Comment by bgruening** [comment]
I'm wondering why it is working in your Docker container. It shouldn't as this package is not installed in Docker and travis is using the same container.

**Comment by cristhomas** [comment]
aaaah, I didn't test in a docker container. maybe that's why. I'm sorry, I didn't realize it would make a difference.

**Comment by bgruening** [comment]
Ah! No big deal. I think if you add this one package and then the build dependency this should be fine :)

**Comment by cristhomas** [comment]
ok let me do that now

**Comment by cristhomas** [comment]
Let me know if this works now

**Comment by bgruening** [comment]
travis will tell you, I need to leave soon, sorry.

**Comment by cristhomas** [comment]
np
thanks for your help

**Comment by bgruening** [comment]
It failed but because of `Error: in section 'source': unknown key 'rpaths'`
I guess your last recipe has something wired in it :)

**Comment by cristhomas** [comment]
argh! neverending =)
I'll fix and check.

**Comment by cristhomas** [comment]
this worked for me in the docker container.

File: recipes/r-geomap/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+package:
+  name: r-geomap
+  version: "2.3_8"
+source:
+  fn: GEOmap_2.3-8.tar.gz
+  url:
+    - http://cran.r-project.org/src/contrib/GEOmap_2.3-8.tar.gz
+    - http://cran.r-project.org/src/contrib/Archive/GEOmap/GEOmap_2.3-8.tar.gz
+  md5: 3207abd337906a9345fc6d2be6c2c212
+  rpaths:
```
Comment (bgruening) [inline recipes/r-geomap/meta.yaml]: This one here is wrong.

`rpath` should not be in source but in `build:`

Comment (cristhomas) [inline recipes/r-geomap/meta.yaml]: thanks for checking. I corrected it. hopefully this was the last mistake.

**Comment by cristhomas** [comment]
hmmm. I didn't do Rgraphviz because it's already in the bioconda recipe. Not sure how to proceed.

**Comment by bgruening** [comment]
Please merge my PR :)
The problem is that some packages are not available on OSX.
This is a general problem and more complicated, so I have skip this for the moment.

**Comment by cristhomas** [comment]
ok!
Didn't I merge your PR already?

**Comment by bgruening** [comment]
There should be a second one, isn't it?

**Comment by bgruening** [comment]
Wuhu! Great! These will be available in 20-30 minutes on anaconda.org

**Comment by cristhomas** [comment]
Awesome, thanks for your help!!

**Pull request merged**